src/plugins/ai_assistant_management/selection/public/routes/components/ai_assistant_selection_page.test.tsx

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import type { CoreStart } from '@kbn/core/public';
+import { AiAssistantSelectionPage } from './ai_assistant_selection_page';
+import { useAppContext } from '../../app_context';
+import { I18nProvider } from '@kbn/i18n-react';
+
+jest.mock('../../app_context');
+
+describe('AiAssistantSelectionPage', () => {
+  const setBreadcrumbs = jest.fn();
+  const navigateToApp = jest.fn();
+
+  const generateMockCapabilities = (hasPersission: boolean) =>
+    ({
+      observabilityAIAssistant: { show: hasPersission },
+      securitySolutionAssistant: { 'ai-assistant': hasPersission },
+    } as unknown as CoreStart['application']['capabilities']);
+
+  const testCapabilities = generateMockCapabilities(true);
+
+  const renderComponent = (capabilities: CoreStart['application']['capabilities']) => {
+    (useAppContext as jest.Mock).mockReturnValue({
+      capabilities,
+      setBreadcrumbs,
+      navigateToApp,
+    });
+    return render(<AiAssistantSelectionPage />, {
+      wrapper: I18nProvider,
+    });
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('sets the breadcrumbs on mount', () => {
+    renderComponent(testCapabilities);
+    expect(setBreadcrumbs).toHaveBeenCalledWith([
+      {
+        text: 'AI Assistant',
+      },
+    ]);
+  });
+
+  it('renders the title and description', () => {
+    renderComponent(generateMockCapabilities(true));
+    expect(screen.getByText('AI Assistant')).toBeInTheDocument();
+    expect(
+      screen.getByText(
+        'AI Assistants use generative AI to help your team by explaining errors, suggesting remediation, and helping you request, analyze, and visualize your data.'
```

Comment:
Any way we can have this not depend on the full text, so it doesn't break with a text change?